apps/vr-tests/convertStoriesToStoryWright.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+#!/usr/bin/env bash
+
+find ./src -type f |while read fname; do
+  impor=""
```

Comment:
impor is a strange name

Reply:
Will address in follow-up PRs with cleanups!